### DISCUSSION TITLE
Adding support for managing resins, pellets, ...?

### DISCUSSION THREAD

**Discussion by lbivens** [Donkie/Spoolman#252] [Ideas]
Hi! As somebody that prints with filament, and resins, I think it would be great to be able to unify the management of materials under spoolman.
I am willing to spend some time into it, if y'all think it fits the project!

**Comment by KnightArch** [comment]
I'll would like to have it too. Maybe @Donkie will consider this in the near future, I hope. I remember to see a repo here with a catalog of every filament, spools, resins and bottles of all over the world. For some reason I didn't mark it and now I have to dig the internet to find it again. It will be very helpful to this kind of feature. I'll bring it here if a manage to find this repository again.

**Comment by ChristophCaina** [comment]
I would support that request, since I am using FDM as well as SLA.
I do see some challenges - especially, if you would mix different resin types... but I think, the basic support should be possible - and then it would be up to you, if you create "custom mixtures" with your resin.

The major difference are the metrics - since you would calculate liquids and volume.

1. Material Type Toggle: A setting to mark a material as "Liquid/Resin".
2. Unit Logic Change: If "Liquid" is selected, hide length calculations (meters) and focus entirely on Weight (g) and Volume (ml).
3. Density as Primary Key: For liquids, the density (g/cm^3) should be the primary factor for volume calculation, ignoring the diameter field.
4. Mixing Support (Advanced): A simple way to create a "Blend" from two existing bottles (e.g., 900g of Bottle A + 100g of Bottle B = New virtual bottle).
5. Bottle Tare Presets: Instead of "Spool weight", use "Bottle tare weight" for common resin bottle sizes (500g, 1000g, 2000g).
